lib/cosmos/packets/packet_config.rb

```diff
@@ -1182,6 +1182,10 @@ def process_current_item(parser, keyword, params)
         begin
           # require should be performed in target.txt
           klass = params[0].filename_to_class_name.to_class
+          unless klass
+            # Try the target namespaced class
+            klass = "Cosmos::#{@current_packet.target_name}::#{params[0].filename_to_class_name}".to_class
```

Comment:
Sounds good. I didn't like the duplication but didn't really know the best way to eliminate it.